tests/guestlog/guestlog.go

```diff
@@ -135,7 +129,7 @@ var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, func()
 				Eventually(func(g Gomega) bool {
 					logs, err := getConsoleLogs(virtlauncherPod)
 					g.Expect(err).ToNot(HaveOccurred())
-					return strings.Contains(logs, cirrosLogo)
+					return strings.Contains(logs, cirrosCheck)
 				}, cirrosStartupTimeout*time.Second, 2*time.Second).Should(BeTrue())
```

Comment:
`.Should(ContainSubstring(cirrosCheck))` is going to be much easier to work with